teamB/JeongWooJin/routes/index.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const router = express.Router();
+
+const userRouter = require("./userRouter");
+router.use("/users", userRouter.router);
+
+const postRouter = require("./postRouter");
+router.use("/posts", postRouter.router);
+
+const likeRouter = require("./likeRouter");
+router.use("/pushLikes", likeRouter.router);
+
+
+module.exports = router;
```

Comment:
코드 자체에는 이상이 없지만, 가독성 및 컨벤션을 위해 다음과 같이 수정하면 좋을 것 같습니다.

1. require은 결국 다른 곳에 짜여진 코드를 불러 오는 것입니다. 그래서 require은 상단에 다 모아 둡니다.
2. 모아 둘 때도 3가지 그룹으로 나눠서 불러오는 일종의 약속 같은 것이 있습니다.
- 첫번째 그룹: node에서 기본적으로 제공하는 모듈
- 두번째 그룹: npm install로 설치한 3rd-part 모듈
- 세번째 그룹: 프로젝트 내에서 작성한 모듈 


```suggestion
const express = require("express");

const userRouter = require("./userRouter");
const postRouter = require("./postRouter");
const likeRouter = require("./likeRouter");

const router = express.Router();

router.use("/users", userRouter.router);
router.use("/posts", postRouter.router);
router.use("/likes", likeRouter.router);
```